src/components/TabQueryEditor.vue

```diff
@@ -466,7 +493,9 @@
           "Ctrl-S": this.triggerSave,
           "Cmd-S": this.triggerSave,
           "Ctrl+Shift+F": this.formatSql,
-          "Cmd+Shift+F": this.formatSql
+          "Cmd+Shift+F": this.formatSql,
+          "Ctrl-C": this.cancelQuery,
+          "Cmd-C": this.cancelQuery
```

Comment:
We need to add a check here to see if the query is running before capturing this keycode because ctrl-c otherwise needs to be `copy`.
Also we don't want cmd-c to be cancel, that would stay ctrl-c (mac is confusing)

Reply:
Oh your right, i just followed the issue suggestion and thought `ctrl+c` makes sense because for terminals its the same shortcut.

What would be a good shortcut for that ?

Reply:
I think ctrl-c is fine, this keybinding can be dynamic!

if (this.running) result['Ctrl+C'] = this.cancelQuery

Reply:
That's the nice thing about vue computed properties :-)

Reply:
Im not sure if i understand what do you mean.

Do you mean a computed property which returns an `object` (with the keymap) if a query is running and if not, it returns an empty object ?
That object will be merged with `runQueryKeyMap` ?

Reply:
I tried the following which doesnt work, because the computed function will be called once not multiple times
```
const runQueryKeyMap = Object.assign({
          "Shift-Ctrl-Enter": this.submitCurrentQuery,
          "Shift-Cmd-Enter": this.submitCurrentQuery,
          "Ctrl-Enter": this.submitTabQuery,
          "Cmd-Enter": this.submitTabQuery,
          "Ctrl-S": this.triggerSave,
          "Cmd-S": this.triggerSave,
          "Shift-Ctrl-F": this.formatSql,
          "Shift-Cmd-F": this.formatSql,
          "Ctrl-/": this.toggleComment,
          "Cmd-/": this.toggleComment
        }, this.cancelQueryKeyMap)
```
and as computed prop
```
cancelQueryKeyMap() {
        if (!this.running) {
          return {};
        }

        return {
          "Ctrl-C": this.cancelQuery,
          "Cmd-C": this.cancelQuery
        }
      }
```